packages/jest-docblock/src/index.ts

```diff
@@ -24,9 +24,19 @@ export function extract(contents: string): string {
   return match ? match[0].trimLeft() : '';
 }
 
+// strip returns contents without docblock
 export function strip(contents: string) {
   const match = contents.match(docblockRe);
-  return match && match[0] ? contents.substring(match[0].length) : contents;
+
+  if (!match) {
+    return contents;
+  }
+
+  if (!match[0]) {
+    return contents;
+  }
+
+  return contents.replace(match[0], '');
 }
```

Comment:
Can someone verify this? 
It seems like the previous version of the strip function was not stripping the docblock correctly because it assumed that the docblock would be at the top of `contents`.

I've updated the function to be a bit more legible

Reply:
this seems sensible to me